src/xcode/ENA/ENA/Source/Scenes/QRScanner/Model/Parser/QRCodeParser.swift

```diff
@@ -71,6 +71,8 @@ class QRCodeParser: QRCodeParsable {
 				healthCertificateService: healthCertificateService,
 				markAsNew: markCertificateAsNew
 			)
+		} else if qrCode.trimmingCharacters(in: .whitespacesAndNewlines).hasPrefix("{") {
```

Comment:
is this the only possibility to check for ticket validation QR codes? And if so, could we wrap this into a constant or an enum?

Reply:
Not really sure if it's worth to put it in a constant, as we don't reuse it and don't have another similar pattern up until now. I implemented it like this because the tech spec says 

> **Check for DCC Ticketing QR code:** if `barcode data` starts with `{`, the `barcode data` shall be parsed as a generic JSON object. If the object has an attribute `protocol` with value `DCCVALIDATION` (case-sensitive), the QR code shall be interpreted as a DCC Ticketing QR code ...

Reply:
then let it as it is